cdc/sink/mysql/mysql_test.go

```diff
@@ -1499,6 +1498,74 @@ func TestExecDMLRollbackErrRetryable(t *testing.T) {
 	require.Nil(t, err)
 }
 
+func TestMysqlSinkNotRetryErrDupEntry(t *testing.T) {
+	errDup := mysql.NewErr(mysql.ErrDupEntry)
```

Comment:
```suggestion
    t.Parallel()

	errDup := mysql.NewErr(mysql.ErrDupEntry)
```

Reply:
Can not parallel, it changes `GetDBConnImpl`, and will affect others tests.

Reply:
Got it!